Latest build crashes

Running 349952a8e5653092e2845fe3d05d15d6d35e2f28.

[Log](http://paste.ubuntu.com/14486118/)

```
W/System.err(22181): 	... 51 more
W/dalvikvm(22181): threadid=1: calling UncaughtExceptionHandler
E/AndroidRuntime(22181): FATAL EXCEPTION: main
E/AndroidRuntime(22181): Process: io.jari.dumpert, PID: 22181
E/AndroidRuntime(22181): java.lang.RuntimeException: Unable to start activity ComponentInfo{io.jari.dumpert/io.jari.dumpert.activities.MainActivity}: android.view.InflateException: Binary XML file line #17: Error inflating class android.support.design.widget.NavigationView
E/AndroidRuntime(22181): 	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2338)
E/AndroidRuntime(22181): 	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2390)
E/AndroidRuntime(22181): 	at android.app.ActivityThread.access$800(ActivityThread.java:151)
E/AndroidRuntime(22181): 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1321)
E/AndroidRuntime(22181): 	at android.os.Handler.dispatchMessage(Handler.java:110)
E/AndroidRuntime(22181): 	at android.os.Looper.loop(Looper.java:193)
E/AndroidRuntime(22181): 	at android.app.ActivityThread.main(ActivityThread.java:5292)
E/AndroidRuntime(22181): 	at java.lang.reflect.Method.invokeNative(Native Method)
E/AndroidRuntime(22181): 	at java.lang.reflect.Method.invoke(Method.java:515)
E/AndroidRuntime(22181): 	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:824)
E/AndroidRuntime(22181): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:640)
E/AndroidRuntime(22181): 	at de.robv.android.xposed.XposedBridge.main(XposedBridge.java:132)
E/AndroidRuntime(22181): 	at dalvik.system.NativeStart.main(Native Method)
E/AndroidRuntime(22181): Caused by: android.view.InflateException: Binary XML file line #17: Error inflating class android.support.design.widget.NavigationView
E/AndroidRuntime(22181): 	at android.view.LayoutInflater.createView(LayoutInflater.java:620)
E/AndroidRuntime(22181): 	at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:696)
E/AndroidRuntime(22181): 	at android.view.LayoutInflater.rInflate(LayoutInflater.java:755)
E/AndroidRuntime(22181): 	at android.view.LayoutInflater.inflate(LayoutInflater.java:492)
E/AndroidRuntime(22181): 	at de.robv.android.xposed.XposedBridge.invokeOriginalMethodNative(Native Method)
E/AndroidRuntime(22181): 	at de.robv.android.xposed.XposedBridge.handleHookedMethod(XposedBridge.java:631)
E/AndroidRuntime(22181): 	at android.view.LayoutInflater.inflate(Native Method)
E/AndroidRuntime(22181): 	at android.view.LayoutInflater.inflate(LayoutInflater.java:397)
E/AndroidRuntime(22181): 	at android.view.LayoutInflater.inflate(LayoutInflater.java:353)
E/AndroidRuntime(22181): 	at android.support.v7.app.AppCompatDelegateImplV7.setContentView(AppCompatDelegateImplV7.java:256)
E/AndroidRuntime(22181): 	at android.support.v7.app.AppCompatActivity.setContentView(AppCompatActivity.java:109)
E/AndroidRuntime(22181): 	at io.jari.dumpert.activities.MainActivity.onCreate(MainActivity.java:53)
E/AndroidRuntime(22181): 	at android.app.Activity.performCreate(Activity.java:5264)
E/AndroidRuntime(22181): 	at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1088)
E/AndroidRuntime(22181): 	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2302)
E/AndroidRuntime(22181): 	... 12 more
E/AndroidRuntime(22181): Caused by: java.lang.reflect.InvocationTargetException
E/AndroidRuntime(22181): 	at java.lang.reflect.Constructor.constructNative(Native Method)
E/AndroidRuntime(22181): 	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
E/AndroidRuntime(22181): 	at android.view.LayoutInflater.createView(LayoutInflater.java:594)
E/AndroidRuntime(22181): 	... 26 more
E/AndroidRuntime(22181): Caused by: android.view.InflateException: Binary XML file line #40: Error inflating class Button
E/AndroidRuntime(22181): 	at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:713)
E/AndroidRuntime(22181): 	at android.view.LayoutInflater.rInflate(LayoutInflater.java:755)
E/AndroidRuntime(22181): 	at android.view.LayoutInflater.inflate(LayoutInflater.java:492)
E/AndroidRuntime(22181): 	at de.robv.android.xposed.XposedBridge.invokeOriginalMethodNative(Native Method)
E/AndroidRuntime(22181): 	at de.robv.android.xposed.XposedBridge.handleHookedMethod(XposedBridge.java:631)
E/AndroidRuntime(22181): 	at android.view.LayoutInflater.inflate(Native Method)
E/AndroidRuntime(22181): 	at android.view.LayoutInflater.inflate(LayoutInflater.java:397)
E/AndroidRuntime(22181): 	at android.support.design.internal.NavigationMenuPresenter.inflateHeaderView(NavigationMenuPresenter.java:190)
E/AndroidRuntime(22181): 	at android.support.design.widget.NavigationView.inflateHeaderView(NavigationView.java:250)
E/AndroidRuntime(22181): 	at android.support.design.widget.NavigationView.<init>(NavigationView.java:172)
E/AndroidRuntime(22181): 	at android.support.design.widget.NavigationView.<init>(NavigationView.java:94)
E/AndroidRuntime(22181): 	... 29 more
E/AndroidRuntime(22181): Caused by: android.content.res.Resources$NotFoundException: File res/drawable/login_ripple.xml from drawable resource ID #0x7f020064
E/AndroidRuntime(22181): 	at android.content.res.Resources.loadDrawable(Resources.java:2152)
E/AndroidRuntime(22181): 	at android.content.res.TypedArray.getDrawable(TypedArray.java:602)
E/AndroidRuntime(22181): 	at android.content.res.XResources$XTypedArray.getDrawable(XResources.java:965)
E/AndroidRuntime(22181): 	at android.view.View.<init>(View.java:3579)
E/AndroidRuntime(22181): 	at android.widget.TextView.<init>
V/Provider/Settings(  695):  from settings cache , name = dropbox:data_app_crash , value = null
D/dalvikvm(  695): create interp thread : stack size=128KB
D/dalvikvm(  695): create new thread
```

--- a/Dumpert/src/main/java/io/jari/dumpert/activities/MainActivity.java
+++ b/Dumpert/src/main/java/io/jari/dumpert/activities/MainActivity.java
@@ -12,6 +12,7 @@ import android.support.design.widget.NavigationView;
 import android.support.v4.view.GravityCompat;
 import android.support.v4.widget.DrawerLayout;
 import android.support.v7.app.ActionBarDrawerToggle;
+import android.support.v7.widget.AppCompatButton;
 import android.support.v7.widget.SearchView;
 import android.support.v7.widget.Toolbar;
 import android.util.Log;
@@ -44,7 +45,7 @@ public class MainActivity extends BaseActivity implements
     private        DrawerLayout      drawer;
     private static SharedPreferences credentials;
     private static NavigationView    navigationView;
-    private static TextView          loginAction;
+    private static AppCompatButton   loginAction;
     private        int               navItemID = R.id.nav_new;
 
     @Override
@@ -164,7 +165,7 @@ public class MainActivity extends BaseActivity implements
         drawer.setDrawerListener(toggle);
         toggle.syncState();
 
-        loginAction = (TextView) navigationView.getHeaderView(0).findViewById(R.id.login_action);
+        loginAction = (AppCompatButton) navigationView.getHeaderView(0).findViewById(R.id.login_action);
         loginAction.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View v) {

--- a/Dumpert/src/main/res/drawable-v21/login_ripple.xml
+++ b/Dumpert/src/main/res/drawable-v21/login_ripple.xml
@@ -1,8 +1,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<ripple
-    xmlns:android="http://schemas.android.com/apk/res/android"
-    android:color="?colorPrimary">
-
-    <item android:drawable="?colorAccent" />
-
-</ripple>

--- a/Dumpert/src/main/res/drawable/login_ripple.xml
+++ b/Dumpert/src/main/res/drawable/login_ripple.xml
@@ -3,6 +3,6 @@
     xmlns:android="http://schemas.android.com/apk/res/android"
     android:shape="rectangle">
 
-    <solid android:color="?colorAccent" />
+    <solid android:color="@android:color/white" />
 
 </shape>

--- a/Dumpert/src/main/res/drawable/login_ripple.xml
+++ b/Dumpert/src/main/res/drawable/login_ripple.xml
@@ -1,8 +1,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<shape
-    xmlns:android="http://schemas.android.com/apk/res/android"
-    android:shape="rectangle">
-
-    <solid android:color="@android:color/white" />
-
-</shape>

--- a/Dumpert/src/main/res/layout/navigation_login.xml
+++ b/Dumpert/src/main/res/layout/navigation_login.xml
@@ -37,10 +37,10 @@
         android:singleLine="true"
         android:textAppearance="@style/TextAppearance.AppCompat.Body1" />
 
-    <Button
+    <android.support.v7.widget.AppCompatButton
         android:id="@+id/login_action"
         android:text="@string/nav_logout"
-        android:background="@drawable/login_ripple"
+        android:background="?attr/selectableItemBackground"
         android:layout_row="0"
         android:layout_column="2"
         android:layout_gravity="end"
@@ -50,6 +50,7 @@
         android:paddingBottom="@dimen/activity_vertical_margin"
         android:paddingLeft="@dimen/activity_horizontal_margin"
         android:paddingRight="@dimen/activity_horizontal_margin"
-        android:textAppearance="@style/TextAppearance.AppCompat.Body1" />
+        android:textAppearance="@style/TextAppearance.AppCompat.Body1"
+        style="@style/Widget.AppCompat.Button.Colored"/>
 
 </GridLayout>

